### DISCUSSION TITLE
Custom Toast Widths

### DISCUSSION THREAD

**Discussion by FTSTimSin** [twilio-labs/paste#2594] [Q&A] [answered]
Hi, I'm trying to use toaster.push to display a toast message, but on smaller devices (mobile), the toast banner is too long and gets cut off. Is there a way to adjust the width of a toast?
![Screen Shot 2022-08-05 at 8 39 17 AM](https://user-images.githubusercontent.com/7245323/183112225-3836e547-dfb1-49d9-bffe-893bc16fc634.png)

**Comment by loreina** [comment, marked as answer]
Hi @FTSTimSin! You can use our [Customization Provider](https://paste.twilio.design/customization/customization-provider/#providing-component-elements) to change the Toast width.

**Reply by SiTaggart** [reply]
Codesandbox demo: https://codesandbox.io/s/customizing-toasts-using-paste-customization-provider-eoy0el?file=/src/index.tsx

**Reply by FTSTimSin** [reply]
Thank you both your your help! Can you provide an example of this being used with a toaster rather than individual toasts? My program uses toaster.push to push toasts.

**Reply by SiTaggart** [reply]
It should be exactly the same way. The Toaster shows and hides Toast components, so targeting the element name in the customization provider, should be sufficient for toaster use cases.

Give it a shot, and let us know if you have any troubles.

**Reply by FTSTimSin** [reply]
Alright I'll give it a shot, thank you so much for your time and help!

**Reply by FTSTimSin** [reply]
@SiTaggart @loreina I originally couldn't get it to work until I realized I had to update Paste haha. Thank you both so much, this worked perfectly!
